diff --git a/node_modules/cloudinary/cloudinary.js b/node_modules/cloudinary/cloudinary.js
index 2f9e02e..0c88616 100644
--- a/node_modules/cloudinary/cloudinary.js
+++ b/node_modules/cloudinary/cloudinary.js
@@ -1,4 +1,4 @@
-if (Number(process.versions.node.split('.')[0]) < 8) {
+if (Number((process.versions.node||"").split('.')[0]) < 8) {
   console.warn('DEPRECATION NOTICE - Node 6 has been scheduled for removal from the Cloudinary SDK');
   // required polyfills for native ES6/7 functions (such as String.prototype.padStart)
   require('core-js');
