diff --git a/node_modules/react-table/src/utils.js b/node_modules/react-table/src/utils.js
index 80bef77..04f252a 100644
--- a/node_modules/react-table/src/utils.js
+++ b/node_modules/react-table/src/utils.js
@@ -12,7 +12,7 @@ export function findMaxDepth(columns, depth = 0) {
 
 // Build the visible columns, headers and flat column list
 export function linkColumnStructure(columns, parent, depth = 0) {
-  return columns.map(column => {
+  return columns && columns.map(column => {
     column = {
       ...column,
       parent,
